src/assets/Standards/ERC20Standard.ts

```diff
@@ -1,6 +1,6 @@
 import { abiERC20 } from '@metamask/metamask-eth-abis';
 import { BN, toUtf8 } from 'ethereumjs-util';
-import { AbiCoder } from 'ethers/lib/utils';
+import { AbiCoder } from '@ethersproject/abi';
```

Comment:
We should add this to the `package.json` too!

Reply:
yea! thought i had!